keps/sig-node/2535-ensure-secret-pulled-images/README.md

```diff
@@ -214,6 +230,8 @@ For alpha, exhaustive Kubelet unit tests will be provided. Functions affected by
 ```
 [TestShouldPullImage link](https://github.com/kubernetes/kubernetes/pull/94899/files#diff-7297f08c72da9bf6479e80c03b45e24ea92ccb11c0031549e51b51f88a91f813R311-R438)
 
+PersistHashMeta()
+
 At beta we should revisit if integration buckets are warranted for e2e node and/or cri-tools/critest, and after gathering feedback.
 
 ### Graduation Criteria
```

Comment:
For PRR

> ###### What happens if we reenable the feature if it was previously rolled back?
>
> Will go back to working as designed.

I think this section should highlight that admin would need to go back to whatever old way they were using to enforce this authn check.  And also, as the feature is rolling out to kubelets (which is slow), they need to retain any API server based checks until rollout has completed.

Reply:
added your comment in..